plugins/generator-1.21.1/neoforge-1.21.1/procedures/utils/projectiles/potion.java.ftl

```diff
@@ -0,0 +1,11 @@
+private static Projectile createPotionProjectile(Level level, ItemStack contents, Entity shooter, Vec3 acceleration) {
+	ThrownPotion entityToSpawn = new ThrownPotion(EntityType.POTION, level);
+	entityToSpawn.setItem(contents);

```

Comment:
This part of the code could be delegated to a call of "plugins/generator-1.21.1/neoforge-1.21.1/procedures/utils/projectiles/throwable.java.ftl"

Maybe it would also make sense to rename "throwable.java.ftl" to "projectile.java.ftl"